ansible_base/resource_registry/service_client.py

```
@@ -0,0 +1,146 @@
+        logger.info(f"Making {method} request to {url}.")
+
+        kwargs = {**self.requests_auth_kwargs, "method": method, "url": url, "verify": self.verify_https}
+
+        if data:
+            kwargs["json"] = data
+        if params:
+            kwargs["params"] = params
+        if stream:
+            kwargs["stream"] = stream
+
+        resp = requests.request(**kwargs)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)